ui/pages/Dashboard/components/ComputedColoredIcon.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import randomMC from 'random-material-color'
+import styled from 'styled-components'
```

Comment:
this is copied form the previous file so does not need review

Reply:
I'm curious that the module `ComputedColoredIcon` doesn't look big. Why should it be a separate lazy-loaded module?

Reply:
because it uses the `randomMC` module which is used nowhere else in the app. Separating out this component removes that node module from the main bundle